src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +28,12 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream().filter(word -> word.length() > 12)
+                .sorted((str1, str2) -> str2.length() - str1.length())
+                .map(String::toLowerCase)
+                .distinct()
+                .limit(100)
+                .forEach(System.out::println);
```

Comment:
```suggestion
        String joinedWords = words.stream().filter(word -> word.length() > 12)
                .map(String::toLowerCase)
                .distinct()
                .limit(100)
                .sorted((str1, str2) -> str2.length() - str1.length())
                .collect(Collectors.joining("\n"));
        System.out.println(joinedWords);
```

데이터를 추린다음 정렬을 하는 것이 성능상 이점이 있습니다.

그리고 표준 출력을 사용할 일이 거의 없겠지만
System.out.println 같은 I/O 작업을 100건 호출하는 방식보다는
문자열을 만들고 한번에 출력하는 것이 더 좋을 것 같아요

Reply:
12와 100도 상수로 치환할 수 있을 것 같아요